.travis.yml

```diff
@@ -20,7 +20,6 @@ before_script:
   - composer install --prefer-dist --no-interaction
 
 script:
-  - sh -c "if [ '$DEFAULT' = '1' ]; then vendor/bin/codecept run; fi"
```

Comment:
Why are we removing tests? Doesnt this one ship at least with very basic instance check tests etc?
You also dont seem to run phpstan level 5+, which both together is really dangerous IMO.

Reply:
@dereuromark You are right. I was confused with running scripts